apps/mark-scan/frontend/src/pages/validate_ballot_page.tsx

```diff
@@ -45,7 +45,7 @@ export function ValidateBallotPage(): JSX.Element | null {
   const invalidateBallotMutation = invalidateBallot.useMutation();
   const validateBallotMutation = validateBallot.useMutation();
   async function invalidateBallotCallback() {
-    invalidateBallotMutation.mutate();
```

Comment:
I think there are some other places that also need this same change (e.g. L53 below). Might be worth just auditing all of the mutations quickly

Reply:
Sorry about that. I updated all state machine mutations to use `mutateAsync` in [1141982](https://github.com/votingworks/vxsuite/pull/3855/commits/1141982f7ace8d2913c4afc5d3ee62ed841972f1). Below is a quick audit of all `.mutate*` calls in the frontend. I've left unrelated mutation calls alone (mainly auth).

```
vx@debian-11:~/code/vxsuite/apps/mark-scan/frontend$ ag '.mutate'
src/app_root.tsx
372:    endCardlessVoterSessionMutation.mutate(undefined, {
399:    await unconfigureMachineMutation.mutateAsync();
461:      startCardlessVoterSessionMutation.mutate(
478:    endCardlessVoterSessionMutation.mutate(undefined, {
500:      await endCardlessVoterSessionMutation.mutateAsync();
605:            await checkPinMutation.mutateAsync({ pin });

src/pages/validate_ballot_page.tsx
48:    await invalidateBallotMutation.mutateAsync();
53:    await validateBallotMutation.mutateAsync();

src/components/session_time_limit_tracker.tsx
19:      logOut={/* istanbul ignore next */ () => logOutMutation.mutate()}
23:          updateSessionExpiryMutation.mutate({ sessionExpiresAt })

src/pages/jam_cleared_page.tsx
30:      endCardlessVoterSessionMutation.mutate(undefined);

src/pages/print_page.tsx
34:    await printBallotMutation.mutateAsync({ pdfData: Buffer.from(pdfData) });

src/pages/admin_screen.tsx
64:    setPrecinctSelectionMutation.mutate({
158:            <SetClockButton logOut={() => logOutMutation.mutate()}>

src/pages/poll_worker_screen.tsx
351:                            await setAcceptingPaperStateMutation.mutateAsync();
378:                          await setAcceptingPaperStateMutation.mutateAsync();

src/pages/unconfigured_election_screen_wrapper.tsx
27:      void configureMutation.mutateAsync().then((result) => {
```

Reply:
Thanks! 

I just realized I missed something in my last comment, which is that it's not totally safe to just change to `await mutation.mutateAsync()` because promise rejections in event handlers will cause uncaught exceptions (which are particularly problematic because they crash Jest with no error message). This was the original motivation to use `.mutate` instead of `.mutateAsync`. The recommended workaround is to use `.mutate` with an `onSuccess` handler. An alternative workaround if that doesn't work is to surround `mutateAsync` with a try/catch. It's a really annoying limitation, but we haven't figured out a better solution yet.

Reply:
Thanks for clarifying! Updated in [bc3d1dc](https://github.com/votingworks/vxsuite/pull/3855/commits/bc3d1dc3516894f0ab99e48d4a611be531fd2aed)

The change to `unconfigureMachine` is later reverted in [9236705](https://github.com/votingworks/vxsuite/pull/3855/commits/92367053270875728f6e892b7920bec5efda67a2) (I'm assuming `main` has the intended usage)